src/test/java/PersonTest.java

```diff
@@ -92,4 +95,129 @@ void getMother(){
         //Assert
         assertEquals(person2, mother);
     }
-}
\ No newline at end of file
+
+    @Test
+    void getChilderen() {
+        //Arrange
+        Person person1 = new Person("Sjaakie", "Breedweg", 34, "Male");
+        Person child1 = new Person("Henkie", "Breedweg", 12, "Male");
+        Person child2 = new Person("Eefje", "Breedweg", 8, "Female");
+        Person child3 = new Person("Bobby", "Breedweg", 10, "Male");
+
+        List<Person> kids = new ArrayList<Person>();
+        kids.add(child1);
+        kids.add(child2);
+        kids.add(child3);
+        person1.setChildren(kids);
+
+        //Act
+        List<Person> children = person1.getChildren();
+
+        //Assert
+        assertEquals(kids, children);
+    }
+
+    @Test
+    void getSiblings(){
+        //Arrange
+        Person person1 = new Person("Sjaakie", "Breedweg", 34, "Male");
+        Person sibling1 = new Person("Suzanne", "Breedweg", 48, "Female");
+        Person sibling2 = new Person("Bebby", "Breedweg", 35, "Female");
+        Person sibling3 = new Person("Dave", "Breedweg", 42, "Male");
+
+        List<Person> sibs = new ArrayList<Person>();
+        sibs.add(sibling1);
+        sibs.add(sibling2);
+        sibs.add(sibling3);
+        person1.setSibling(sibs);
+
+        //Act
+        List<Person> sibling = person1.getSibling();
+
+        //Assert
+        assertEquals(sibs, sibling);
+    }
+
+    @Test
+    void addParents(){
+        //Arrange
+        Person person1 = new Person("Sjaakie", "Breedweg", 34, "Male");
+        Person person2 = new Person("Henk", "Richard", "Breedweg", 68, "Male");
+        Person person3 = new Person("Beppy", "Breedweg", 63, "Female");
+
+        //Act
+        person1.addParents(person2, person3);
+        Person mother = person1.getMother();
+        Person father = person1.getFather();
+
+        //Assert
+        assertEquals(person2, father);
+        assertEquals(person3, mother);
+    }
+
+    @Test
+    void addSibling(){
+        //Arrange
+        Person person1 = new Person("Sjaakie", "Breedweg", 34, "Male");
+        Person sibling1 = new Person("Suzanne", "Breedweg", 48, "Female");
+        List<Person> sibs = new ArrayList<>();
+        sibs.add(sibling1);
+
+        //Act
+        person1.addSibling(sibling1);
+
+        //Assert
+        assertEquals(sibs, person1.getSibling());
```

Comment:
Goed opgezette test, netjes met de AAA-structuur. 

Volgens mij kan het wel zo zijn dat de volgorde van de lijst verandert bij meerdere siblings. En in dat geval, kan assertEquals een andere volgorde geven. 

In jouw test is dat niet aan de orde, omdat er maar 1 sibling is. Maar om de test robuuster te maken, kan je ook kiezen voor assertTrue(person1.getSibling().contains(sibling1));

Wat daarnaast ook vaak voorkomt, is dat de lengte van de lijst wordt gecheckt: assertEquals(1, person1.getSlibing().size()); Zo weet je zeker dat er 1 sibling is en dat die sibling de juiste persoon is